BootloaderCommonPkg/Library/IasImageLib/IasImage.c

```diff
@@ -68,7 +68,7 @@ IsIasImageValid (
 
   if (!FeaturePcdGet (PcdVerifiedBootEnabled)) {
     DEBUG ((DEBUG_INFO, "IAS image verification is skipped!\n"));
-    return Hdr;
+    return (IAS_HEADER *) ImageAddr;
   }

```

Comment:
How it could solve any issue since Hdr is set to (IAS_HEADER *) ImageAddr in line 49, they are exactly same.

Reply:
ImageAddr is the input pointer instead of local variable so it should resolve the issue right? Local pointer will be deleted when exiting the function.

Reply:
Same question here.  The change seems to be equivalent to the original code.

Reply:
I think the fix should 
change:
  IasImageInfo->HashData = DigestHash;
to:
  CopyMem (IasImageInfo->HashData, DigestHash, sizeof(IasImageInfo->HashData));

Reply:
Guys I've confused about the input pointer and local pointer. Will update the pull request once again.

Reply:
There is no DigestHash, is the KW based on old code?

Reply:
I guess the code built in the server is out dated, I will check again at KW side.